notebooks/notebooks/iclr2022/baseline-attacks.ipynb

```diff
@@ -15,125 +15,405 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        "mirror": colors[2],
</pre><p>Is there a reason the keys are in this order? If we moved "mirror" down one line then the indexes would be in ascending order which is a bit easier to read, but perhaps we want to present mirror earlier in the legend?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>